SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialTypebindingManager.h

```diff
@@ -84,7 +84,7 @@ class SPATIALGDK_API USpatialTypebindingManager : public UObject
 	// Returns true if the class path corresponds to an Actor or Subobject class path in SchemaDatabase
 	bool IsSupportedClass(UClass* Class) const;
 
-	FClassInfo* FindClassInfoByClass(UClass* Class);
+	FClassInfo& FindClassInfoByClass(UClass* Class);
 	FClassInfo* FindClassInfoByActorClassAndOffset(UClass* Class, uint32 Offset);
```

Comment:
I had refactored all other FindClassInfo functions to return a reference too with assertions for the nullptr cases. Do you think I should try to do this after applying your changes?

Reply:
The other `FindClassInfo` functions will return nullptr in some cases, and the caller will handle that case, you can't just make them return by ref without adding some logic on the call sites.